x-pack/plugins/observability_solution/dataset_quality/public/components/flyout/flyout_summary/flyout_summary_kpi_item.tsx

```diff
@@ -71,8 +63,7 @@ export function FlyoutSummaryKpiItem({
                 alignItems: 'center',
                 width: 'fit-content',
               }}
-              href={link.href}
-              target="_blank"
+              {...link.props}
```

Comment:
Removed `target="_blank"` to make it consistent with how other links work (e.g. on main page).